src/FSharp.Compiler.LanguageServer/FSharp.Compiler.LanguageServer.fsproj

```diff
@@ -0,0 +1,52 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<OutputType>Exe</OutputType>
+		<TargetFramework>net8.0</TargetFramework>
+		<CheckNulls>true</CheckNulls>
+		<Nullable>enable</Nullable>
+	</PropertyGroup>
+
+	<ItemGroup Label="Package References">
+		<PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="8.0.0" />
+		<PackageReference Include="Microsoft.VisualStudio.LanguageServer.Protocol" Version="17.13.6-preview" />
+		<PackageReference Include="Microsoft.VisualStudio.Threading" Version="17.12.21" />
+		<PackageReference Include="StreamJsonRpc" Version="2.21.3-preview" />
```

Comment:
Don't forget to move those to variables from `versions.props`

Reply:
I hope to do that while figuring out the vsix situation, because I will be messing around with the props files anyway then